robot/web/stream/server.go

```diff
@@ -245,6 +254,8 @@ func (ss *Server) AddStream(ctx context.Context, req *streampb.AddStreamRequest)
 func (ss *Server) RemoveStream(ctx context.Context, req *streampb.RemoveStreamRequest) (*streampb.RemoveStreamResponse, error) {
 	ctx, span := trace.StartSpan(ctx, "stream::server::RemoveStream")
 	defer span.End()
+	ss.logger.Warnf("RemoveStream START %s", req.Name)
+	defer ss.logger.Warnf("RemoveStream END %s", req.Name)
```

Comment:
you've got a few logger messages here and there (I'm assuming they're for debugging as you work on this), just a heads up to remove them before you  merge.

Reply:
yeah I'm in the progress of debugging